stubs/pyOpenSSL/@tests/stubtest_allowlist.txt

```diff
@@ -1,4 +0,0 @@
-OpenSSL.SSL.Connection.__init__
-OpenSSL.SSL.Context.load_verify_locations
-OpenSSL.SSL.Context.set_verify
-OpenSSL.crypto
```

Comment:
stubtest complains that `OpenSSL.crypto` doesn't exist. If you don't want to figure out what's going on with that, you can leave `OpenSSL.crypto` in the allowlist for this PR.

Reply:
It seems I've figured out what's going on and proposed a fix:
https://github.com/python/mypy/pull/10663